build.gradle

```diff
@@ -90,7 +90,7 @@ dependencies {
     // Custom Libraries
     compile 'uk.gov.ons.census.fwmt:census-fwmt-canonical:1.0.17-RELEASE'
     compile 'uk.gov.ons.census.fwmt:census-fwmt-events:1.0.23-RELEASE'
-    compile 'uk.gov.ons.census.fwmt:census-fwmt-common:1.1.0-RELEASE'
+    compile 'uk.gov.ons.census.fwmt:census-fwmt-common:1.1.0-SNAPSHOT'
```

Comment:
there are the following versions in artifactory
1.1.0-RELEASE
1.1.1-RELEASE
1.1.2-RELEASE
change to one of these